src/milia/api/project.cljc

```diff
@@ -97,6 +97,14 @@
   (let [url (make-url "projects" id "star")]
     (parse-http :delete url)))
 
+(defn toggle-star
+  "Toggle between starred and unstarred for a user's project"
+  [id star? callback]
+  (let [url  (make-url "projects" id "star")]
+    (parse-http (if star? :post :delete)
```

Comment:
making this call here is duplicating the function of `add-star` and `remove-star`, can we make callback an optional arg in `add-star` and `remove-star` (and in this fn) then vary calling `add-star` or `remove-star` here, based on `star?`.

To make `callback` an optional it's best to use extra arg destructuring, so changing the signature to something like:

```clojure
(defn toggle-star
  [id star? & {:keys [callback]}]
  ...
```